crates/muvm/src/bin/muvm.rs

```diff
@@ -383,58 +362,59 @@ fn main() -> Result<ExitCode> {
     }
 
     let muvm_guest_path = find_muvm_exec("muvm-guest")?;
-    let muvm_server_path = find_muvm_exec("muvm-server")?;
 
-    let mut muvm_guest_args: Vec<String> = vec![
+    let display = env::var("DISPLAY").ok();
+    let guest_config = GuestConfiguration {
+        command: Launch {
```

Comment:
If I'm reading the code correctly, only `command` and `command_args` is actually used, right? Or is this intentional so that the user can pass in more stuff in the muvm config file in the future?

Reply:
Yes, that is correct, and somewhat intentional, i intend to make the first and the following commands no longer behave differently in the future